### DISCUSSION TITLE
Keyboard Tab in modal doesn't work when modal is opened from a menu-item

### DISCUSSION THREAD

**Discussion by vboechat** [chakra-ui/chakra-ui#7418] [Q&A] [answered]
I'm trying to navigate in my modal using the TAB key from my keyboard, but when I try this in a modal opened from a menu-item, it doesn't work.
I checked navigation with TAB in a modal that is opened from a normal Button and it works normally, so the problem is not my modal component, but the menu / menu-item Chakra UI component.

How can I fix this glitch?

**Comment by romanstetsyk** [comment]
@vboechat I've just encountered the same behavior. Looks like a bug to me! Have you been able to find a solution yet?

**Reply by vboechat** [reply]
nope, I didn't find a solution yet :/

**Reply by romanstetsyk** [reply]
I've just created an [issue](https://github.com/chakra-ui/chakra-ui/issues/7697). Let's see if it gets some attention

**Comment by romanstetsyk** [comment, marked as answer]
@vboechat and everyone who stumbled upon this issue, [here](https://github.com/chakra-ui/chakra-ui/issues/7697#issuecomment-1566217291) is the answer.
